client/router.js

```diff
@@ -452,15 +452,15 @@ Router.prototype._initTriggersAPI = function() {
   this.triggers = {
     enter: function(triggers, filter) {
       triggers = Triggers.applyFilters(triggers, filter);
-      _.extend(self._triggersEnter, triggers);
+      if(triggers.length) self._triggersEnter = self._triggersEnter.concat(triggers);
```

Comment:
Try to add brackets to `if`. See: https://github.com/meteorhacks/code-standards/blob/master/docs/style_guide.md#brace-style